src/utils/testbotuser.py

```diff
@@ -0,0 +1,7 @@
+class TestBotUser:
```

Comment:
I like the idea of adding algorithmic bots, that maybe just play random safeties or random cards. Let's extend this idea in a future PR and implement bot strategies for different games, and have a "beat the bot" mode etc.